stream-chat-android-client/build.gradle

```diff
@@ -69,6 +69,7 @@ dependencies {
     implementation Dependencies.androidxSecurityCrypto
     implementation Dependencies.workRuntimeKtx
     implementation Dependencies.constraintLayout
+    implementation Dependencies.androidxRecyclerview
```

Comment:
Can we do something to avoid adding this dependency to our client? I don't like it being here just for debug purposes. We can probably get away with just using a `ScrollView` for this dialog or something.

Reply:
Yes, changed it to using old friendly `ListView`